tests/butlerAlias.py

```diff
@@ -65,7 +65,7 @@ def getDefaultLevel(self):
         return 'visit'
 
     def getKeys(self, datasetType, level):
-        return {'filter': types.StringType, 'visit': types.IntType}
+        return {'filter': bytes, 'visit': int}
```

Comment:
why bytes here and not str? (seems wrong, but maybe it's a new python3 thing I don't know about yet?)